### PR TITLE
ENG-1789: Adds new section (DW+DSP)

### PR DESCRIPTION
**Pull request by bredamc**
First draft of the new section:
"Running distributed data science workloads from data science pipelines"

### CODE REVIEW

File: modules/running-distributed-data-science-workloads-from-ds-pipeline.adoc [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+----
+$ python compile_example.py
+----
+This command creates a YAML file (in this example, `compiled-example.yaml`), which you can import in the next step.
+. Import your data science pipeline as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-with-data-science-pipelines_ds-pipelines#importing-a-data-science-pipeline_ds-pipelines[Importing a data science pipeline].
+. Schedule the pipeline run as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-with-data-science-pipelines_ds-pipelines#managing_pipeline_runs[Scheduling a pipeline run].
+. When the pipeline run is complete, view the run details as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-with-data-science-pipelines_ds-pipelines#viewing-the-details-of-a-pipeline-run_ds-pipelines[Viewing the details of a pipeline run].
+
+.Verification
+The YAML file is created and the pipeline run completes without errors.
```
Comment (chtyler) [inline modules/running-distributed-data-science-workloads-from-ds-pipeline.adoc:124]: Would a record of this run be included in the "Triggered pipeline runs" interface?

File: modules/running-distributed-data-science-workloads-from-ds-pipeline.adoc [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+.. Compile the Python file (in this example, the `compile_example.py` file):
++
+[source,bash]
+----
+$ python compile_example.py
+----
+This command creates a YAML file (in this example, `compiled-example.yaml`), which you can import in the next step.
+. Import your data science pipeline as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-with-data-science-pipelines_ds-pipelines#importing-a-data-science-pipeline_ds-pipelines[Importing a data science pipeline].
+. Schedule the pipeline run as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-with-data-science-pipelines_ds-pipelines#managing_pipeline_runs[Scheduling a pipeline run].
+. When the pipeline run is complete, view the run details as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-with-data-science-pipelines_ds-pipelines#viewing-the-details-of-a-pipeline-run_ds-pipelines[Viewing the details of a pipeline run].
```
Comment (chtyler) [inline modules/running-distributed-data-science-workloads-from-ds-pipeline.adoc:121]: Essentially this action would verify that the run was completed. By viewing the details of the run to examine its details, it is in-turn being verified.

File: modules/running-distributed-data-science-workloads-from-ds-pipeline.adoc [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+.Verification
+The YAML file is created and the pipeline run completes without errors.
+
+////
+[role='_additional-resources']
+.Additional resources
+<Do we want to link to additional resources?>
```
[the remark this answers is not in the document]
Comment (chtyler) [reply modules/running-distributed-data-science-workloads-from-ds-pipeline.adoc:129]: Along with links to any Ray cluster docs too.

File: modules/running-distributed-data-science-workloads-from-ds-pipeline.adoc [hunk 4]
```diff
@@ -0,0 +1,133 @@
+:_module-type: PROCEDURE
+
+[id="running-distributed-data-science-workloads-from-ds-pipeline_{context}"]
+= Running distributed data science workloads from data science pipelines
+
+[role='_abstract']
+To run a distributed data science workload from a data science pipeline, you must first update the pipeline to include a link to your Ray cluster image.
+
+.Prerequisites
+* You have access to a data science cluster that is configured to run distributed workloads as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-with-distributed-workloads_distributed-workloads#configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
```
Comment (chtyler) [inline modules/running-distributed-data-science-workloads-from-ds-pipeline.adoc:10]: They need to be logged in to the dashboard to do most of the tasks in the procedure.

**Comment by bredamc** [comment]
@chtyler Many thanks for your review comments, draft updated accordingly :)

**Review by dimakis — APPROVED** [review]
LGTM, excellent work as ever!

**Review by chtyler — APPROVED** [review]

**Pull request merged**